ci/dev/validate_json.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
```

Comment:
Not for this PR but given this validation will be used across repos I'm wondering if it could be moved to the recipes to avoid duplication.

Reply:
That's a good point! moving the validation to a module should help. Will take a look.